<launch>

    <!-- <rosparam command="load" file="$(find swipe_obstacles)/cfg/swipe_obstacles.yaml"/> -->
    <node pkg="ras" type="ras_core_node" name="core"/>
    <node pkg="ras" type="ras_autoware_connector_node" name="autoware_connector"/>
    <node pkg="ras" type="ras_visualizer_node" name="visualizer"/>
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find ras)/cfg/ras.rviz"/>

</launch>
